### ISSUE TITLE
MGit crashing on my device

### ISSUE THREAD

**Issue by Lourens-Rich** [maks/MGit#373]
Hi. I will get down to business.

I have MGit from F-Droid. I tryed pull updates from my repo and then app was crashed.

Here is log:

```
USER_COMMENT=
ANDROID_VERSION=7.0
APP_VERSION_NAME=1.5.4.4
BRAND=HUAWEI
PHONE_MODEL=SLA-L22
CUSTOM_DATA=
STACK_TRACE=java.lang.ClassCastException: me.sheimi.sgit.activities.RepoDetailActivity cannot be cast to com.manichord.mgit.repolist.RepoListActivity
	at com.manichord.mgit.dialogs.ExceptionDialog.onCreateDialog(ExceptionDialog.kt:23)
	at android.support.v4.app.DialogFragment.onGetLayoutInflater(DialogFragment.java:328)
	at android.support.v4.app.Fragment.performGetLayoutInflater(Fragment.java:1226)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1428)
	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1759)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1827)
	at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2596)
	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2383)
	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2338)
	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2245)
	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:703)
	at android.os.Handler.handleCallback(Handler.java:761)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:156)
	at android.app.ActivityThread.main(ActivityThread.java:6595)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:832)
```

**Comment by Lourens-Rich** [comment]
Sorry. I don't read another issues. It's https://github.com/maks/MGit/issues/367?

**Comment by inatz16** [comment]
good

**Comment by maks** [comment]
@Lourens-Rich yes it is the same issue as #367 and so should be fixed in 1.5.4.5 which I've just pushed out via Google Play for all users.
